hello_nexus/basic/README.md

```diff
@@ -0,0 +1,37 @@
+This sample shows how to define a Nexus service, implement the operation handlers, and
+call the operations from a workflow.
+
+### Sample directory structure
+
+- [service.py](./service.py) - shared Nexus service definition
+- [caller](./caller) - a caller workflow that executes Nexus operations, together with a worker and starter code
+- [handler](./handler) - Nexus operation handlers, together with a workflow used by one of the Nexus operations, and a worker that polls for both workflow and Nexus tasks.
+
+
+### Instructions
+
+Start a Temporal server.
+
+Run the following:
+
+```
+temporal operator namespace create --namespace my-handler-namespace
+temporal operator namespace create --namespace my-caller-namespace
```

Comment:
We support providing namespace at `start-dev` time, that may be easier for users